third_party/terraform/utils/config.go.erb

```diff
@@ -143,161 +143,161 @@ func (c *Config) LoadAndValidate() error {
 	c.userAgent = userAgent
 
 	log.Printf("[INFO] Instantiating GCE client...")
-	c.clientCompute, err = compute.New(client)
+	c.clientCompute, err = compute.NewService(context.Background(), option.WithHTTPClient(client))
```

Comment:
We should instantiate `context.Background` a single time and pass the shared object to all these clients. This will allow us to create a context tree that can allow cancelling 1 or all requests when the parent is cancelled.

Reply:
Not sure it does anything- we never cancel the parent anyways, and most requests end pretty soon- but done.